src/scss/use-cases/_wrapper.scss

```diff
@@ -5,71 +5,60 @@
 
 /// General typography
 /// Apply to a wrapper to style all text inside it
-///
+/// @breaking Remove 'style' param, default to product headings, move article headings out of o-typography.
+/// @example scss
+///     article { @include oTypographyWrapper; }
 /// @brand master
 /// @param {String} $style will change the headings in a body that is not an article
-/// @example scss
-/// article { @include oTypographyWrapper; }
-/// non-article { @include oTypographyWrapper($style: product); }
 @mixin oTypographyWrapper($style: article) {
-	@if $style == article {
+
+	// Only master brand supports product and article header styles.
```

Comment:
remind me, what happens to internal headings if they don't fall under 'product' ?

Reply:
The default headers are now sans for the internal brand -- previously they were unusable for the internal as they were serif article headings. Now the internal brand has usable default headings, the product headings are deprecated for the internal brand -- until they are removed, they look the same as they do now.